qcodes/instrument_drivers/QuTech/IVVI.py

```diff
@@ -183,7 +183,45 @@ def _get_version(self):
             mes = self.ask(bytes([3, 4]))
             ver = mes[2]
             return ver
-
+    
+    def linsweep(self,start,end,points):
+        """
+        A replacement of numpy.linspace for use with the ivvi rack dacs.
+        ivvi.linsweep gives:
+         - even spacing in voltages
+         - voltages match the set voltages
+         - protection from oversampling
+         
+        Args:
+            start (number): starting voltage
+            end (number): ending voltage
+            points (int): number of points between start and end, inclusive
+            
+        Returns:
+            list of valid voltages for the ivvi rack
+            
+        Examples:
+            #normal dac aligned voltages
+            ivvi.linsweep(-200,200,201) -> [ -199.404, -197.391 ... ] 
+            
+            #protection from oversampling
+            ivvi.linsweep(start=108.8,end=109,points=11) -> 
+                [108.82734416723888, 108.88838025482566,108.94941634241245, 109.01045242999923]
+        """
+        points = max(round(abs(points)),2)
+        use_reversed = end < start
+        if use_reversed:
+            start,end = end,start
+        dac_quata = 4.0/65535.0
```

Comment:
this is clearly a magic number that needs to be defined as a constant (in the module, or at least as a class attribute). Are the values of `4.0` (i assume this is some range) and `65535.0` (i assume this is some bit depth/resolution) the same for all the ivvi's? Even if yes, they deserve to be declared as constants so that it is clear what they mean and what role they play in this method.